### DISCUSSION TITLE
Is the `@react-aria/collections` inside `react-aria-components` stable enough to be used in production?

### DISCUSSION THREAD

**Discussion by miridih-yhkim** [adobe/react-spectrum#6805] [Q&A] [answered]
Hello,

I am considering using either the `react-stately/collection` or the `@react-aria/collections`, which is used in react-aria-components.

However, I noticed that `@react-aria/collections` is still in the alpha stage, which is quite confusing given that `react-aria-components` is at version 1.3.1.

According to the documentation at https://react-spectrum.adobe.com/Support.html#alpha, alpha versions are considered unstable. Does this mean that `@react-aria/collections` is not suitable for production yet?

**Comment by snowystinger** [comment, marked as answer]
Thanks for the question and for reading the docs. It's fine to use. When we say unstable in this case, we mean that we haven't quite settled the API yet. So as long as you're ok with keeping up with API changes, the behaviors are pretty stable. We've been using it now for over a year developing RAC.

**Reply by miridih-yhkim** [reply]
Oh, thank you for the quick response. 

As i understood, `@react-aria/collections` is already stable enough, but it remains in the alpha stage because more features and test coverage are planned?

**Reply by snowystinger** [reply]
More tests are always planned ;)
It's in alpha stage while we gather feedback on the API and usage.
